logrotate.c

```diff
@@ -1286,7 +1290,7 @@ static int findNeedRotating(struct logInfo *log, int logNum, int force)
                 }
                 break;
             case ROT_DAYS:
-                /* FIXME: only days=1 is implemented!! */
+                /* TODO: only days=1 is implemented!! */
```

Comment:
I fail to see how TODO fits better here than FIXME.  The comment is there at least since 1996 and I have not seen any requests by logrotate's users to implement it.  I think it should be safe to just remove the comment.